docs/using-avail/getting-started/explorer.md

```diff
@@ -283,18 +279,6 @@ medium.
 
 :::
 
-## Receive AVL Testnet Tokens
-
-Head over to the [faucet](https://faucet.polygon.technology).
-
-On the faucet page, select `DA (Test Token)` and `DA Network` as the token and network,
-respectively. Paste your account address and click on **Submit**. The transfer will take up to one
-minute to complete.
-
-<img src={useBaseUrl("img/avail/faucet.png")} width="100%" height="100%"/>
-
-Upon successful transfer, your account should now have a non-zero balance. If you face any issues obtaining tokens from the faucet, please reach out on [Discord](https://discord.gg/y6fHnxZQX8).
-
 ### Balance Check with `@polkadot/api`
```

Comment:
We should probably mention version here, or point to some wrapper like `avail-js-api`.

Reply:
Yes.

Ideally it will just point to `avail-js-api` but I want to take some time to make sure that API is what we want before we advertise it.

What should we say about @polkadot/api version in the meantime? Use 10.3.4?

This page also has other issues too and could use some reworking or even a rewrite—I just moved it over from the old docs.

Reply:
This section was actually commented out. I deleted it for now.